src/XMakeTasks/Exec.cs

```diff
@@ -466,7 +466,7 @@ protected override bool ValidateParameters()
 
             // if the working directory is a UNC path, and all drive letters are mapped, bail out, because the pushd command
             // will not be able to auto-map to the UNC path
-            if (workingDirectoryIsUNC && (DriveInfo.GetDrives().Length == 26))
+            if (workingDirectoryIsUNC && NativeMethods.AllDrivesMapped())
```

Comment:
does `workingDirectoryIsUNC` even have meaning on Unix?  Should this be resolved by making that always false?